tests/psimulate/worker/test_vivarium_work_horse.py

```diff
@@ -21,7 +30,19 @@ def test_setup_sim(mocker):
         results_path="~/tmp",
         extras={},
     )
-    sim_config = setup_sim(job_parameters).configuration
 
-    assert sim_config["time"]["end"]["year"] == 2020
-    assert sim_config["input_data"]["artifact_path"] == "~/vivarium.yaml"
+    initial_job_params = job_parameters.branch_configuration.copy()
+    update_dict = parameter_update_format(job_parameters)
+    sim_config = setup_sim(job_parameters).configuration.to_dict()
+    job_config = job_parameters.branch_configuration
+
+    def compare_dicts(test_dict, ref_dict):
```

Comment:
This is basically checking that every key in the `ref_dict` exists in the `test_dict` and is matches the `ref_dict` value? I think it'd be good to add a comment to state that explicitly, because I originally read this and expected it to be checking the that the dicts are equal, which this is not doing.